ci/pipeline.yml

```diff
@@ -109,6 +110,35 @@ jobs:
       file: bundled-deps/bundled-deps-*.tgz
   on_failure: #@ slack_failure_notification()
 
+- name: test-e2e
+  serial: true
+  plan:
+  - in_parallel:
+    - { get: repo, trigger: true }
+    - { get: pipeline-tasks }
+    - { get: bundled-deps, tags: ["galoy-staging"], trigger: true}
+  - task: test-e2e
+    timeout: 12m
+    tags: ["galoy-staging"]
+    config:
+      platform: linux
+      image_resource: #@ task_image_config()
+      inputs:
+      - name: bundled-deps
+      - name: pipeline-tasks
+      - name: repo
+        path: galoy
```

Comment:
We need to give this a different path so that the `docker compose` prefixes on the docker machine don't collide.
This should probably be `galoy-e2e` and the integration test job can be updated to `galoy-integration` path.

Reply:
Not quite sure what you mean by this, don't they both have the same repository input with `path: galoy`

Reply:
They do - which is a problem because the path determines the scope of the `docker compose up` container naming. If they have the same path the 2 contexts will interfere with each other

Reply:
We have a shared singleton docker vm where things get run - its not isolated between concourse jobs